index.yaml

```diff
@@ -40,6 +40,15 @@ pages:
     file: contributing.md
     description: How to contribute to Ignition.
 releases:
-  - citadel
-  - blueprint
-  - acropolis
+  - name: citadel
+    lts: true
+    eol: false
+    description: Recommended version.
+  - name: blueprint
+    lts: false
+    eol: false
+    description: Works with Ubtuntu 18.04.
```

Comment:
This one can be misleading, because we're also supporting Citadel on 18.04. But I don't have a good suggestion for an alternative.

* "Ignition's first T-Shirt" :laughing: 
* "Supported for 1.5 years"
* "Works with ROS Melodic"